pages/1.13/security/ent/tls-ssl/exhibitor/index.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: layout.pug
+navigationTitle:  Securing Exhibitor with mutual TLS
+title: Securing Exhibitor with mutual TLS
+menuWeight: 500
+excerpt: Securing DC/OS with a TLS enabled Exhibitor ensemble
+enterprise: true
+---
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+By default, the Exhibitor HTTP service is open to any client that can reach port 8181 on a master node. This documentation chapter describes a method for protecting the Exhibitor service from unauthorized access. Once enabled, HTTP clients must access Exhibitor through Admin Router; thus applying the Admin Router access control policy to the exhibitor service.
+
+*Note: When accessing Exhibitor through Admin Router (**https://master_host/exhibitor**), authenticated users must have the `dcos:adminrouter:ops:exhibitor` privilege*
+
+# Securing Exhibitor
+
+The strategy for securing exhibitor is mutual TLS authentication. In order to secure exhibitor we must first create a unique root CA. This CA must be used to sign a client certificate for each master node. Creating a public key infrastructure that outputs PEM and Java Keytool formatted artifacts is not a trivial task. To make this processes easier, a simple tool has been created for producing the necessary files. 
```

Comment:
Let's generally capitalize Exhibitor instead of writing exhibitor.

Reply:
"a unique root CA certificate." instead of "a unique root CA."

Reply:
Maybe "to sign various end entity certificates for each master node" instead of "to sign a client certificate for each master node"?

(after all, we create client certificates and server certificates :-))